.eslintrc

```diff
@@ -17,7 +17,8 @@
         "react/no-find-dom-node": 0,
         "flowtype/define-flow-type": 1,
         "flowtype/use-flow-type": 1,
-        "no-color-literals": 1
+        "no-color-literals": 1,
+        "prefer-const": [2, {"destructuring": "all"}],
```

Comment:
Could you switch this from error (2) to warning (1). We fail both in CI, but only error during builds, which can be annoying for things that aren't critical errors.

Or maybe we should treat all errors as warnings in `build-watch` and `build-hot`?